reference/json/BigCommerce_Orders_API.oas2.json

```diff
@@ -2365,22 +2365,18 @@
           "example": "S2549JM0Y",
           "type": "string"
         },
-        "amount": {
-          "description": "Amount of the discount. This information is returned as in integer. Dollar and percentage discounts will return the same. \nFor example, $3 returns as '3' while 5% will return as 5. Check the discount type to see what type of discount is available. (Float, Float-As-String, Integer)",
-          "example": "5.0000",
-          "type": "string"
-        },
         "type": {
           "type": "string",
           "title": "Order Coupon Discount Type",
+          "description": "type 0: per_item_discount\ntype 1: percentage_discount\ntype 2: per_total_discount\ntype 3: shipping_discount\ntype 4: free_shipping\ntype 5: promotion",
           "enum": [
             0,
             1,
             2,
             3,
-            4
-          ],
-          "description": "type 0: per_item_discount\ntype 1: percentage_discount\ntype 2: per_total_discount\ntype 3: shipping_discount\ntype 4: free_shipping"
+            4,
+            "5"
```

Comment:
Should this 5 be wrapped in quotes? all the preceding numbers are not.

Reply:
Yes, it needs to be an int like the rest.